decidim-core/lib/decidim/core/test/factories.rb

```diff
@@ -600,6 +600,17 @@ def generate_localized_title
         some_extra_data: "1"
       }
     end
+
+    trait :comment_notification do
+      event_class { "Decidim::Comments::CommentCreatedEvent" }
+      event_name { "decidim.events.comments.comment_created" }
+      extra { { comment_id: create(:comment).id } }
+    end
+
+    trait :proposal_notification do
```

Comment:
Can we move the trait definition to the only spec using it?

Reply:
done 👌